studio/components/interfaces/Organization/OAuthApps/AuthorizeRequesterDetails.tsx

```diff
@@ -1,13 +1,68 @@
+import { OAuthScope } from '@supabase/shared-types/out/constants'
+import { IconCheck } from 'ui'
+
 export interface AuthorizeRequesterDetailsProps {
   icon: string | null
   name: string
   domain: string
+  scopes: OAuthScope[]
+}
+
+const ScopeSection = ({
+  description,
+  readScope,
+  writeScope,
+  allScopes,
```

Comment:
just wondering - can we simplify this? we just need `hasRead` and `hasWrite` in this component right? should we just make those the arguments instead? then we don't need to pass `allScopes` as well - reduce the amount of props

`const ScopeSection = ({ description, hasRead, hasWrite }) => ...`